{
  try {
    mojo.execute();
    Assert.fail("Expected MojoFailureException not thrown");
  }
 catch (  MojoFailureException e) {
    Assert.assertEquals(message,e.getMessage());
  }
}
